pkg/kubefedctl/enable/enable.go

```diff
@@ -286,6 +286,16 @@ func CreateResources(cmdOut io.Writer, config *rest.Config, resources *typeResou
 		}
 
 		for _, ftc := range ftcs.Items {
+			targetAPI := concreteTypeConfig.Spec.TargetType
+			existingAPI := ftc.Spec.TargetType
+			if IsEquivalentAPI(&existingAPI, &targetAPI) {
+				existingName := fmt.Sprintf("%s.%s/%s", existingAPI.PluralName, existingAPI.Group, existingAPI.Version)
+				name := fmt.Sprintf("%s.%s/%s", targetAPI.PluralName, targetAPI.Group, targetAPI.Version)
+
+				return errors.Errorf("Target API %q of FederatedTypeConfig %q is equivalent to existing target API %q of FederatedTypeConfig %q. You can disable existing resource before enabling the new one",
```

Comment:
-> `("Failed to enable %q. Federation of this type is already enabled for equivalent type %q by FederatedTypeConfig %q", name, existingName, qualifiedFTCName)`

e.g. 

`Failed to enable "deployment.extensions/v1beta1". Federation of this type is already enabled for equivalent type "deployments.apps/v1" by FederatedTypeConfig "kube-federation-system/deployments.apps"`